.github/workflows/csharp_client_compatibility.yaml

```diff
@@ -37,24 +40,29 @@ jobs:
         version: ${{ fromJson(needs.setup_server_matrix.outputs.matrix) }}
         kind: [os, enterprise]
         os: [ ubuntu-latest, windows-latest ]
+
     name: Test CSharp ${{ github.event.inputs.branch_name }} branch against ${{ matrix.kind }} ${{ matrix.version }} server on ${{ matrix.os }}
     steps:      
       - name: Checkout to scripts
         uses: actions/checkout@v4
+
       - name: Setup Python
         uses: actions/setup-python@v2
         with:
           python-version: 3.9
+
       - name: Install .NET
         uses: actions/setup-dotnet@v3
         with:
           dotnet-version: |
-            2.1.x
-            3.1.x
-            5.0.x
-            6.0.x
```

Comment:
why removed?

Reply:
Net 8 covers those, and <6.0 are out of support anymore.